r/src/array_to_vector.cpp

```diff
@@ -960,6 +960,18 @@ bool ArraysCanFitInteger(ArrayVector arrays) {
   return all_can_fit;
 }
 
+bool GetBoolOption(const std::string& name, bool default_) {
+  SEXP getOption = Rf_install("getOption");
+  cpp11::sexp call = Rf_lang2(getOption, Rf_mkString(name.c_str()));
+  cpp11::sexp res = Rf_eval(call, R_BaseEnv);
+  Rf_PrintValue(res);
```

Comment:
Is this a debugging print statement you meant to remove?
```suggestion
```

Reply:
oops